src/auditor/cli.py

```diff
@@ -38,24 +43,66 @@ def cli():
         print(__doc__)
     else:
 
-        # report_dir = args['--save_report']
+        #: Logger that will gather the summary information.
+        summary_logger = logging.getLogger(__name__)
+        summary_logger.setLevel(logging.DEBUG)
 
-        cli_logger = logging.getLogger('auditor')
-        cli_logger.setLevel(logging.DEBUG)
-        detailed_formatter = logging.Formatter(
-            fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
-        )
-        cli_handler = logging.StreamHandler(stream=sys.stdout)
-        cli_handler.setLevel(logging.DEBUG)
-        cli_handler.setFormatter(detailed_formatter)
-        cli_logger.addHandler(cli_handler)
+        if args['spot']:
 
-        org_auditor = Auditor(cli_logger, args['--verbose'], args['ITEM'])
+            #: Only dump summary info to the console if verbose
+            #: Note: currently, org-wide checks (duplicate names) only reported if verbose
```

Comment:
**question**: why did you decide that duplicate names are only displayed in verbose?

Reply:
It was kind of a bodge to fit it in- the main checks/fixes happen on an item-by-item basis, while the duplicate check happens outside of that loop. This makes it difficult in the current engineering to get the results of that check added to the overall report file. Again, a refactoring is definitely called for.